Clarify creating a custom module (#164)
catppuccin/tmux: README.md

It is possible to add a new custom module or overwrite any of the existing modules.

> Look into [custom/README.md](https://github.com/catppuccin/tmux/blob/main/README.md) for more details.

Reviewer: ```suggestion
For further details, see the documentation in [custom/README.md](custom/README.md)
```
